example/src/Examples/Gooey/Gooey.tsx

```diff
@@ -74,7 +74,7 @@ export const Gooey = () => {
   const transforms = useMemo(
     () =>
       icons.map((icon) =>
-        createComputedValue(
+        ValueApi.createComputedValue(
```

Comment:
Is this related to the Mock? Otherwise it might not be needed in this PR?

Reply:
It is related in the sense that the import path of createComputedValue had React Native dependency. Indeed this is a breaking change and it could potentially be taken aside of this PR but it might not be worth the trouble. Your call.